web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx

```diff
@@ -64,13 +64,11 @@ export const FireBehaviourAdvisoryPage: React.FunctionComponent = () => {
     DateTime.now().setZone('UTC-7').toISO()
   )
 
-  const [previouslySelectedDateOfInterest, setPreviouslySelectedDateOfInterest] =
-    useState(DateTime.now().setZone('UTC-7').toISO())
-
-  const updateDate = () => {
-    if (previouslySelectedDateOfInterest !== dateOfInterest) {
-      setPreviouslySelectedDateOfInterest(dateOfInterest)
-    }
+  const updateDate = (date: string) => {
+    /*This needs to be done in order for timezone to function correctly and not have the 
+    date picker display the incorrect date*/
+    date = `${date}T00:00:00-08:00`
```

Comment:
Please use constant instead of hard -08:00 in string. There's a -8 constant somwhere.

Reply:
Still using a string here